### PR TITLE
Implement initial volume slider

### PR DESCRIPTION
**Pull request by Liberatys**
This pull requests implements a working version of a volume slider. It was one of the pain points when using the application :D. As I was unsure about the location, I just put it somewhere where it could be easily accessed and thus not intrude on the rest of the application.

The slider allows values between 0 - 100 and displays the values as integer in the label (but is chosen as f32).

<details>
<summary>Image of the sidebar</summary>

screen

</details>

I so far had no issues with it and it seems to perform well.

### CODE REVIEW

File: psst-gui/src/controller/playback.rs [hunk 1]
```diff
@@ -222,6 +224,11 @@ impl PlaybackController {
         self.send(PlayerEvent::Command(PlayerCommand::Seek { position }));
     }
 
+    fn update_volume(&mut self, volume: f64)  {
+        self.send(PlayerEvent::Command(PlayerCommand::SetVolume { volume: ((volume / 100.0) as f32) }));
```
Comment (Liberatys) [inline psst-gui/src/controller/playback.rs:228]: The slider widget implements Widget<f64> and I then convert it to f32 for miniaudio

Comment (Liberatys) [reply psst-gui/src/controller/playback.rs:228]: Also we have to divide by 100 as the values is chosen in 0 - 100 but miniaudio is expecting 0 - 1

File: psst-gui/src/controller/playback.rs [hunk 2]
```diff
@@ -40,6 +41,7 @@ impl PlaybackController {
             thread: None,
             output_thread: None,
             media_controls: None,
+            volume: 50.0,
```
Comment (Liberatys) [inline psst-gui/src/controller/playback.rs:44]: I think 50 is a good default?

**Comment by jpochyla** [comment]
Great, thanks!

- I'm not sure about the location. The best we could do, I guess, is a on-hover popup with a vertical slider, somewhere near the playback controls? Hovers are a bit difficult in Druid now, though.
- We don't need to duplicate the `volume` value in the `PlaybackController`, we can diff it in `update`.
- Please please use `rustfmt` for the PRs.

I'll fix the code stuff, we can figure out where to put the slider later.

File: psst-gui/src/controller/playback.rs (hunk 2 above)
Comment (jpochyla) [reply psst-gui/src/controller/playback.rs]: I think that as the default used to be 100, we should keep it.

**Comment by jpochyla** [comment]
I've ended up tweaking the visuals a bit and polishing the code. If you're OK with it we can merge.

**Comment by Liberatys** [comment]
Your solution is much cleaner 👍🏻 nice

**Pull request merged**